modules/clients/src/test/java/org/apache/ignite/jdbc/JdbcAuthorizationTest.java

```diff
@@ -164,7 +169,11 @@ public void testCopyFrom() throws Exception {
         assertAuthorizationFailed(sql, CACHE_READ_USER);
         assertAuthorizationFailed(sql, CACHE_REMOVE_USER);
 
+        int cnt = ignite(0).cache(DEFAULT_CACHE_NAME).size();
```

Comment:
It seems that we should check TEST_BULKLOAD_CACHE cache size here and check its size using equality.

Reply:
Fixed.